rascaline/src/calculators/lode/radial_integral/spline.rs

```diff
@@ -63,6 +64,40 @@ impl LodeRadialIntegralSpline {
             center_contribution: radial_integral.compute_center_contribution()
         });
     }
+
+    pub fn from_tabulated(
+        parameters: LodeRadialIntegralSplineParameters,
+        spline_points: Vec<SplinePoint>,
+        center_contribution: Vec<f64>,
+    ) -> Result<LodeRadialIntegralSpline, Error> {
+
+        let spline_parameters = SplineParameters {
+            start: 0.0,
+            stop: parameters.cutoff,
+            shape: vec![parameters.max_angular + 1, parameters.max_radial],
+        };
+
+        let mut new_spline_points = Vec::new();
+        for spline_point in spline_points {
+            new_spline_points.push(
+                HermitSplinePoint{
+                    position: spline_point.position,
+                    values: spline_point.values.0.clone(),
+                    derivatives: spline_point.derivatives.0.clone(),
+                }
+            );
+        }
+
+        assert_eq!(
```

Comment:
I think this should be a `InvalidParameterError`, `assert_eq` will `panic`, which creates an `internal error (this is a bug)` error message